src/SumatraPDF.cpp

```diff
@@ -2001,30 +2002,18 @@ static void RerenderEverything()
 
 void GetFixedPageUiColors(COLORREF& text, COLORREF& bg)
 {
-    if (gGlobalPrefs->useSysColors) {
-        text = GetSysColor(COLOR_WINDOWTEXT);
-        bg = GetSysColor(COLOR_WINDOW);
-    }
-    else {
-        text = gGlobalPrefs->fixedPageUI.textColor;
-        bg = gGlobalPrefs->fixedPageUI.backgroundColor;
-    }
+    

```

Comment:
I think this (and GetEbookUiColors) should have a toggle to use either the current theme's colours or default (black and white). This toggle should be global though, not individual for ebookUI and fixedPageUI.